app/test_main.py

```diff
@@ -1,3 +1,31 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_convert_to_0_if_age_less_then_15() -> None:
+    goals = get_human_age(7, 7)
+    assert goals == [0, 0]
```

Comment:
```suggestion
    assert get_human_age(7, 7) == [0, 0]
```
do not create extra variables

please, fix everywhere